app/src/main/java/org/oppia/android/app/databinding/ImageViewBindingAdapters.java

```diff
@@ -43,6 +46,27 @@ public static void setImageDrawable(
     imageView.setImageResource(drawableResourceId);
   }
 
+  /**
+   * Allows binding drawables to an [ImageView] via "android:src".
+   * Reference: https://stackoverflow.com/a/35809319/3689782.
+   */
+  @BindingAdapter("android:src")
+  public static void setImageDrawable(
+      @NonNull ImageView imageView,
```

Comment:
Should remove 2 spaces here & below per https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation.

Reply:
I tried doing that but as soon as I press `Delete/Backspace` it goes to previous line. Mostly I think we will need to update the `Preference -> Code Style` but that means some other `.idea` files will also get changed.

Can you check if this works on your end without any issues?

https://user-images.githubusercontent.com/9396084/112985892-6cab3b00-917e-11eb-814f-434368d2bb15.mov

Reply:
Hmm I suspect I'll run into the same issue--you might just need to fix it manually. I'm assuming code style isn't configured correctly for Java development in this codebase yet.

Reply:
Reverted to original code as push fails with following error: `has incorrect indentation level 4, expected level should be 6`